extensibility/PreCommitHooks/MyTransactor.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Starcounter.Database;
+using Starcounter.Database.Hosting.Extensibility;
+using Starcounter.Database.ChangeTracking;
+using Microsoft.Extensions.Options;
+using System.Linq;
+
+class MyTransactor : TransactorBase
+{
+    class MyContext : ContextBase
+    {
+        public MyContext(IDatabaseContext context) : base(context) { }
+
+        public void ExecutePreCommitHooks(PreCommitHookOptions options)
+        {
+            foreach (var change in ChangeTracker.Changes.Where(c => c.ChangeType != ChangeType.Delete))
+            {
+                var proxy = Get<object>(change.Id);
```

Comment:
I have missed this in the [Support getting changes from transaction #1043](https://github.com/Starcounter/Starcounter.Nova/pull/1043/files#diff-4d30b62c07a44d5df78eee29e8d076bfR13) pull request, but I think the `Change.Id` property shall be called `Change.Oid` for the sake of consistency.